editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -96,10 +97,14 @@ void Polygon2DEditor::_notification(int p_what) {
 
 			b_snap_grid->set_icon(get_editor_theme_icon(SNAME("Grid")));
 			b_snap_enable->set_icon(get_editor_theme_icon(SNAME("SnapGrid")));
-			uv_icon_zoom->set_texture(get_editor_theme_icon(SNAME("Zoom")));
 
 			uv_vscroll->set_anchors_and_offsets_preset(PRESET_RIGHT_WIDE);
 			uv_hscroll->set_anchors_and_offsets_preset(PRESET_BOTTOM_WIDE);
+			// Avoid scrollbar overlapping.
```

Comment:
When were they overlapping though?

Reply:
At the corner:
![kuva](https://github.com/godotengine/godot/assets/1621768/25a76274-3724-4e10-993f-d160e341713f)
This code isn't new, I just moved it to more reasonable place (it was at the end of `_uv_draw()`).

Reply:
Ok I thought it's new and didn't see any overlapping in vanilla build.